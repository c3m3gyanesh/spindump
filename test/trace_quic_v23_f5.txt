A test between picquic client and the f5 QUIC server in the network. Commands that were used:

    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 f5quic.com 4433 /5000
    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 f5quic.com 4433 /50000
