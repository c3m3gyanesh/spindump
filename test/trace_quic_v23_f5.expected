QUIC 172.16.114.251 <-> 40.118.202.119 08f1f2945d5e2727-456d224cd4153409 (57515:4433) at 1571184401874180 new starting packets 0 0 bytes 0 0
QUIC 172.16.114.251 <-> 40.118.202.119 08f1f2945d5e2727-0814228ba18548c8 (57515:4433) at 1571184401882698 measurement up right 8518 packets 1 0 bytes 1280 0 bandwidth 1280 0
QUIC 172.16.114.251 <-> 40.118.202.119 087d178755d4a1cb-a0fa174227b53c25 (64187:4433) at 1571184403991098 new starting packets 0 0 bytes 0 0
QUIC 172.16.114.251 <-> 40.118.202.119 087d178755d4a1cb-08cda5192c270e73 (64187:4433) at 1571184403998910 measurement up right 7812 packets 1 0 bytes 1280 0 bandwidth 1280 0
